src/lib/pipes/index.js

```diff
@@ -1,8 +1,10 @@
 // import TestPipe from './test-pipe.js';
 import NormaliseEventPipe from './normalise-event-pipe.js';
+import NormaliseSlotPipe from './normalise-slot-pipe.js';
 
 export default [
   // Comment this out to see some test normalised events in the system
   //TestPipe
-  NormaliseEventPipe
+  NormaliseEventPipe,
```

Comment:
Think this and #84 will have a merge confilct so you should pick which one you want in first :-)